src/tools/dart-tool.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "dart: The Dart command-line tool"
+description: "The reference page for using 'dart'."
```

Comment:
Could we consider including the word `terminal` here for those that search by that work?